src/app/components/media/MediaComponentRightPanel.js

```diff
@@ -37,8 +37,9 @@ const MediaComponentRightPanel = ({
             label={
               <FormattedMessage
                 id="mediaComponent.requests"
-                defaultMessage="Requests"
+                defaultMessage="{requests_count, plural, =0 {Requests} other {Requests [#]}}"
```

Comment:
Please let's not use plurals for this and rather display count programatically. This formatting is not parsed by Transifex.

The formatting accepted by transifex is: `{total, plural, one {# tag} other {# tags}}` which seems to defeat the purpose of what you're trying to accomplish here.

Reply:
I don't think I understand @amoedoamorim I grabbed my code from another part of the app, so I thought it was ok. Regardless, the only difference I see between my code and your formatting is:
- the use of [] around the number
- and the use of the =0 operator

Is one of those two things the issue?

Reply:
this is what I used as the "starting" point:
https://github.com/meedan/check-web/blob/257c58eb1372e93f32e762e3d8c7c2c3ec7d72ef/src/app/components/user/UserTooltip.js#L125

Reply:
Yes, the issue seems to be with the =0 operator, it looks like transifex will only support `one` and `other` for english. Having `=0` in the string causes the pluralized string not be parsed correctly by Transifex and then making it more difficult for translators to write translations correctly.
This pattern has been used since the very beginning of Check localization but this issue was flagged recently by one of the translators and I ended forgetting to submit a fix. This PR just brought this issue back to my attention. 
 I'll send a PR for updating all instances with =0. Sorry for the confusion.

Reply:
Nice, thanks for the info. I will update this PR to match the other pattern